tests/std/tests/P0898R3_concepts/test.cpp

```diff
@@ -1449,7 +1454,11 @@ namespace test_constructible_from {
     };
     STATIC_ASSERT(!test<Multiparameter>());
     STATIC_ASSERT(test<Multiparameter, int>());
-    STATIC_ASSERT(!test<Multiparameter, long>() || is_permissive);

```

Comment:
Note: The removal of `is_permissive` is intentional. It was added in #533 and AFAIK is no longer useful.